webpack/assets/javascripts/react_app/components/common/BreadcrumbSwitcher/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Popover,
+  Button,
+  Icon,
+  ListGroup,
+  ListGroupItem,
+  OverlayTrigger,
+} from 'patternfly-react';
+import { ButtonToolbar } from 'react-bootstrap';
+import EllipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import PropTypes from 'prop-types';
+import './BreadcrumbSwitcher.scss';
+
+const BreadcrumbSwitcher = ({ resources }) => {
+  const popover = (
+    <Popover id="popover-contained">
+      <ListGroup className="scrollable-list">
+        {resources.map((item) => {
+          const { onClick, url, caption } = item;
+          return (
+            <ListGroupItem
+              className="no-border"
+              key={`id-${caption}`}
+              href={url}
```

Comment:
will this work with turbolinks?

Reply:
yes, `ListGroupItem` renders `<a>` tag when href exists.